src/PopOver/index.js

```diff
@@ -122,6 +122,7 @@ PopOver.propTypes = {
 	placement: PropTypes.oneOf(placements),
 	/** Control whether the popover should be positioned using "position: absolute" or "position: fixed" */
 	positionFixed: PropTypes.bool,
+	// eslint-disable-next-line jsdoc/check-indentation
```

Comment:
I've just fixed this on `v2` so this comment shouldn't be needed anymore. JSDoc was complaining about the extra space in front of "Should" in line 128.